server/internal/core/application/covenantless.go

```diff
@@ -1097,6 +1111,15 @@ func (s *covenantlessService) finalizeRound() {
 		return
 	}
 
+	go func() {
+		s.paymentEventsCh <- RoundPaymentEvent{
+			RoundTxID:             round.Txid,
+			SpentVtxos:            getSpentVtxos(round.Payments),
+			SpendableVtxos:        s.getNewVtxos(round),
+			ClaimedBoardingInputs: boardingInputs,
+		}
+	}()
```

Comment:
this should be done in [updateVtxoSet](https://github.com/ark-network/ark/blob/master/server/internal/core/application/covenantless.go#L1220-L1266) (same for covenant)

Reply:
dont have there boardingInputs?